assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -142,6 +148,37 @@ export const DEFAULT_NOTIFICATIONS = {
 		},
 		isDismissible: false,
 	},
+	setup_error: {
+		Component: SetupErrorNotification,
+		priority: 150,
```

Comment:
The IB suggests:

> For priority, bump it by 10 from the last added error notification (they start from 150)

```suggestion
		priority: 160,
```

Reply:
Thanks, good spot

Although on that note, come to think of it, we will need to re-think the priority values. @jimmymadon We are refactoring notifications from the lower priority to the higher priority (from bottom up), and we are starting the error notifications from priority `150`, it was intended to go up from there as I recall, but our queue will sort this in reverse way - lower priority notifications will be order in the top, example:
`authentication-error` - priority `150`, location before deletion - `bottom`, based on the old rendering where css was hiding all items except from `first-child`
`setup_error` - priority `160`,  location before deletion - above `authentication-error` (figuratively saying, don't recall if they were both in `header`), if both of them were to show, setup error would go first in the old system, in the new infrastructure, they would be ordered [`150`, `160`], so to speak order would be reversed from originally intended one and `authentication-error` would surface before `setup_error` one.

Perhaps we should set the range for errors, like 300-150, and start from `300` and decrease by `10` for each newely refactored banner?

Reply:
@zutigrm You are correct, we do have to put the priority below 150, so we can use 140, 130, etc for the other errors.  The IB here was mistaken. When I refactored Gathering and Zero data, I used 150 and 160 because there were about 4-5 more error notifications before, so the errors would technically start from 100 or 110 onwards (which was my rough plan). This is fine.